src/main/java/com/redhat/cloud/common/clowder/configsource/ClowderConfigSource.java

```diff
@@ -307,15 +326,80 @@ public String getValue(String configKey) {
                     if (root.endpoints == null) {
                         throw new IllegalStateException("No endpoints section found");
                     }
-                    String requestedEndpoint = configKey.substring(CLOWDER_ENDPOINTS.length());
-                    for (EndpointConfig endpoint : root.endpoints) {
-                        String currentEndpoint = endpoint.app + "-" + endpoint.name;
+                    String requestedEndpointConfig = configKey.substring(CLOWDER_ENDPOINTS.length());
+                    String[] configPath = requestedEndpointConfig.split("\\.");
+
+                    String requestedEndpoint;
+                    String param;
+                    final String FORMAT_EXAMPLE = "[endpoint-name].[url|trust-store-path|trust-store-password|trust-store-type]";
+                    if (configPath.length == 1) {
+                        log.warn("Endpoint '" + requestedEndpointConfig + "' is using the old format. Please move to the new one: " + FORMAT_EXAMPLE);
```

Comment:
I wonder if we should throw an exception here. That kind of issue would require an immediate fix.

Reply:
I think a warn is fine - it's falling back to the old behavior.